requirements.txt

```diff
@@ -1,5 +1,5 @@
 https://github.com/wazo-platform/xivo-bus/archive/master.zip
-https://github.com/wazo-platform/xivo-lib-python/archive/master.zip
+https://github.com/wazo-platform/xivo-lib-python/archive/WAZO-2364-unicode-error.zip
```

Comment:
can you update the description of the PR, just to tell there is a dependency with xivo-lib-python (not 'master' branch) ?

Reply:
Done

Reply:
As you don't use `txtsubst()` in xivo-lib-python anymore, you don't need to put a dependency to the other PR (https://github.com/wazo-platform/xivo-lib-python/pull/101)

Reply:
You can revert this modification, you can now use the `master` branch of xivo-lib-python, right ?

Reply:
yes, I did that...

Reply:
Yes the `Depends-On` if you are ok of course. 
Because you can now use the `master` branch of xivo-lib-python, right?

Reply:
yes sure, I will remove it :)

Reply:
Done